pontoon/insights/static/js/insights_tab.js

```diff
@@ -104,58 +164,55 @@ var Pontoon = (function (my) {
                   '--status-unreviewed',
                 ),
                 pointHoverBorderColor: style.getPropertyValue('--white-1'),
+                fill: true,
+                tension: 0.4,
               },
             ],
           },
           options: {
-            legend: {
-              display: false,
-            },
-            tooltips: {
-              borderColor: style.getPropertyValue('--status-unreviewed'),
-              borderWidth: 1,
-              caretPadding: 5,
-              xPadding: 10,
-              yPadding: 10,
-              displayColors: false,
-              callbacks: {
-                label: (item) => nf.format(item.value) + ' days',
+            plugins: {
+              tooltip: {
+                borderColor: style.getPropertyValue('--status-unreviewed'),
+                borderWidth: 1,
+                caretPadding: 5,
+                padding: {
+                  x: 10,
+                  y: 10,
+                },
+                displayColors: false,
+                callbacks: {
+                  label: (context) => nf.format(context.parsed.y) + ' days',
+                },
               },
             },
             scales: {
-              xAxes: [
-                {
-                  type: 'time',
-                  time: {
-                    displayFormats: {
-                      month: 'MMM',
-                    },
-                    tooltipFormat: 'MMMM YYYY',
-                  },
-                  gridLines: {
-                    display: false,
-                  },
-                  ticks: {
-                    source: 'data',
+              x: {
+                type: 'time',
+                time: {
+                  displayFormats: {
+                    month: 'MMM',
                   },
+                  tooltipFormat: 'MMM YYYY',
```

Comment:
This (and other instances of the same value) should remain `MMMM YYYY`, which will show the full month name in the tooltip, where we have enough space.